packages/docs/docs/concepts/progressive-loading/requirements.md

```diff
@@ -0,0 +1,41 @@
+---
+id: requirements
+---
+
+# Requirements and Configuration for Progressive Loading
+
+The progressive loading will improve stack image display just given support
+of HTJ2K progressive resolution encoded data, while volumetric data is improved
+in time to first volume for all back ends not using customized streaming loaders.
+However, the support of different types of reduced resolution and streaming
+responses is quite varied between DICOMweb implementations. Thus, this
+guide provides some additional details on how to configure various options,
+as well as how to modify the default load order for different requirements.
+
+## Separate Path or Argument Partial Resolution
+
+One way of configuration for partial image resolution loading is to add
+a separate path. This is done with the framesPath argument, which replaces
```

Comment:
We should talk in the context of WADO. I understand that for local static WADO, you can create a separate path. However, considering that people use commercial PACS systems, this separate path (frame path) might not be feasible at all. At least some discussion about this is needed here.